.buildkite/pipeline.elastic-agent-binary-dra.yml

```diff
@@ -1,3 +1,61 @@
+env:
+  DRA_PROJECT: "elastic-agent-core"
+  DRA_ARTIFACT_SET: "agent-core"
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - group: ":beats: DRA Elastic-Agent Core Snapshot :beats:"
```

Comment:
Why do we need to have a different group per workflow. 

We could calculate the workflow inside the `build-agent-core.sh` depending on the conditions for the environment variables. That way this whole pipeline would be simplified into 2 steps, one would be the build and the other would be the publishing